Keyboards/KeyboardsBase/LanguageDBManager.swift

```diff
@@ -250,7 +250,7 @@ extension LanguageDBManager {
       word = ?
     """
     let args = [word]
-    let outputCols = ["suggestion_0", "suggestion_1", "suggestion_2"]
+    let outputCols = ["autosuggestion_0", "autosuggestion_1", "autosuggestion_2"]
```

Comment:
Nice catch on these! I honestly don't remember when the change for this was made 🤔 Maybe is coming from changes in Scribe-Data's directory structure or something like that :)